rules-topics/16_runecarving.md

```diff
@@ -0,0 +1,104 @@
+
+
+### Tier VII
+
+#### Rune of Elemental Barrier
+   >This rune will protect the character from one instance of Elemental damage, of a type chosen upon application. Only one Elemental Barrier Rune can be applied to an individual at a time. This takes up a defense slot. It takes 5 seconds to apply the Rune. The defensive call for this is "Resist".
+#### Rune of Mirror Shield
+>This creates a temporary Rune that can be applied to a Shield with 1 Minute of Roleplay that will allow the character to Reflect Spell. Only one Shield Spikes or Mirror Shield Rune can be on a shield at a time. The call for this is "Reflect". This does take up a defense slot. This item is consumed on use. 
+#### Rune of Shield Spikes
+   >This creates a temporary Rune that can be applied to a Shield with 1 Minute of Roleplay that will allow the character to Reflect Physical. Only one Shield Spikes or Mirror Shield Rune can be on a shield at a time. The call for this is "Reflect". This does take up a defense slot. This item is consumed on use.
+
+### Tier VIII
+
+#### Elemental Channeling Rune
+   >This Rune is a potent conduit for Elemental forces, that will allow the user to shoot out balls of condensed elemental energy once activated. The player will call "Activate Elemental Channeling" when this item is used. The character may then throw "5 Magic <Element>", chosen from Flame/Frost/Shock/Stone at activation, for as long as the character can maintain concentration, and not move their feet. These packets are physical attacks. This item is consumed once used.
+#### Glyph of Haven
+> A Glyph of Haven wraps a structure with an intangible field that prevents creatures without a Metabolism from entering the structure. Creatures with Undead or No Metabolism may step into the Haven, though they may still attempt to strike creatures across the threshold, or cast spells into the structure. Creatures in the structure with Undead or No Metabolism at time of placement are not forced out, but if they exit they may not re-enter. A structure may only be under the effects of a single Glyph at any time, and a Glyph affects an entire structure, as such, they cannot be "nested" inside one another. A structure under the effects of a Glyph must have each door/entry clearly and obviously display an H for Haven and the Glyph of Haven tag must be attached to the outside of the container with the date and time of activation. Glyphs are not affected by Dispel Magic or Nullify Magic and may not be destroyed via a Destroy effect. A Glyph lasts 5 days. This item is consumed on use.
```

Comment:
` Creatures with Undead or No Metabolism may step into the Haven` Should this be `may not`?

Clarification question for structures with more than one door.  Do they need to mark all doors?

Reply:
"A structure under the effects of a Glyph must have _each door/entry_ clearly and obviously display"